user.exception=user_failure

hero.exception=hero_failure

user.hero.exception=user_hero_failure